conveyal/analysis-ui
# Additional logic for analysis panel

## Description

This PR includes a few small follow-up changes to df538d8d, hiding form elements when they are not applicable based on the selected modes.

## How to test

1. Disable transit. Check that egress mode selector is disabled and inactive.
2. Check that Walk Speed and Bike Speed form elements are shown only when the corresponding mode is selected as an access/direct/egress mode.
2. Check that Max Walk Time and Max Bike Time are shown only when transit is enabled.
